pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -116,15 +116,15 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "openblas";
-  version = "0.3.13";
+  version = "0.3.15";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "xianyi";
     repo = "OpenBLAS";
     rev = "v${version}";
-    sha256 = "14jxh0v3jfbw4mfjx4mcz4dd51lyq7pqvh9k8dg94539ypzjr2lj";
+    sha256 = "1qjr02cqncv20abdp1yzr55n7smhx6h9chqvb0xbp18byynvj87w";
   };
 
   # apply https://github.com/xianyi/OpenBLAS/pull/3060 to fix a crash on arm
```

Comment:
it looks like this issue was already patched, so we need to remove this patch

Reply:
also, can you change the commit message format? https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes is the standard.